target/linux/ipq40xx/base-files/lib/upgrade/platform.sh

```diff
@@ -115,6 +115,7 @@ platform_do_upgrade() {
 	cilab,meshpoint-one |\
 	edgecore,ecw5211 |\
 	edgecore,oap100 |\
+	engenius,eap1300 |\
```

Comment:
This looks wrong. Device has NOR flash yet You try to use NAND upgrade procedure. Does the upgrade not work currently? Later in that file eap1300 is handled by asterisk which should suffice.